components/Elevation/src/UIColor+MaterialElevation.h

```diff
@@ -0,0 +1,40 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ Provides extension to UIColor for Material Elevation usage.
+ */
+@interface UIColor (MaterialElevation)
+
+/**
+ Returns a color that takes the specified elevation value into account.
+ @param elevation The elevation to use when resolving the color.
+ */
+- (UIColor *)mdc_resolvedColorWithElevation:(CGFloat)elevation;
+
+/**
+ Returns a color that takes the specified elevation value and traits into account.
```

Comment:
Because `resolvedColorWithTraitCollection` is used internally and it doesn't document which traits are used and how. I added the documentation to explain the relationship between userStyle and elevation.

Reply:
Addressed in 0cdfbe81803033b128f749ab3e4f4dc30445053a.